README.md

```diff
@@ -2,20 +2,445 @@
 
 [![CircleCI](https://circleci.com/gh/magento-research/pwa-buildpack.svg?style=svg&circle-token=a34631f6c22f0bdd341f9773895f9441584d2e6a)](https://circleci.com/gh/magento-research/pwa-buildpack)
 
-#### Build and development tools for Magento Progressive Web Apps
+Build and development tools for Magento Progressive Web Apps
 
-### Intro
+## Quick Setup
```

Comment:
@jcalcaben to confirm, is this the stuff you're moving to devdocs? Seems slightly out of place in this repo

Reply:
@DrewML 
Yes, this content is moving to the pwa-devdocs repo.
See related Pull Request: https://github.com/magento-research/pwa-devdocs/pull/20